src/database/models.py

```diff
@@ -3186,3 +3186,41 @@ def full_json(self):
     class Meta:
         db_table = "deployments"
         ordering = ("-version",)
+
+
+class FeatureFlag(models.Model):
+    """
+    A class used to represent Feature flags to turn on for
+    communities and users
+    """
```

Comment:
please add some comments on the attributes.  What is owner, audience, user_audience, etc.
The migration file in the PR doesn't reflect this version of the model.